src/MoBi.Core/Services/SimulationLoader.cs

```diff
@@ -110,36 +101,16 @@ private IMoBiSimulation cloneSimulation(IMoBiSimulation simulation)
       private void addSimulationConfigurationToProject(IMoBiSimulation simulation, ICommandCollector commandCollector)
       {
          var config = simulation.Configuration;
-         config.ModuleConfigurations.Each(moduleConfiguration => commandCollector.AddCommand(new AddModuleCommand(moduleConfiguration.Module)));
+         config.ModuleConfigurations.Each(moduleConfiguration => commandCollector.AddCommand(new AddModuleCommand(_context.Clone(moduleConfiguration.Module))));
          
          addToProject(commandCollector, config.Individual, individual => new AddIndividualBuildingBlockToProjectCommand(individual));
          config.ExpressionProfiles.Each(expressionProfile => addToProject(commandCollector, expressionProfile, ep => new AddExpressionProfileBuildingBlockToProjectCommand(ep)));
```

Comment:
isn't it the same problem here? with the other BB? I am not getting it. Shoudn't we clone all and ADD everything then?